coalib/output/ConsoleInteraction.py

```diff
@@ -321,7 +321,7 @@ def print_affected_files(console_printer,
     :param console_printer: Object to print messages on the console.
```

Comment:
it seems the commit message is still not perfect. first of all, youre not mentioning where that docstring is. it should at least be something like: ``ConsoleInteraction: ..`` or ``Console:`` 

also, in commit body, dont use past tense, use present simple, as in: "Changes".